distribution/lib/Standard/Base/0.0.0-dev/src/Errors/Common.enso

```diff
@@ -49,6 +54,18 @@ type Type_Error
        - name: The name of the argument whose type is mismatched.
     Error expected actual name
 
+    ## PRIVATE
+       Infer the type of the actual value in a human-readable format.
+    type_of_actual self =
+        case (Meta.type_of self.actual) of
+            Compile_Error.Error _ -> self.actual.to_display_text
```

Comment:
Wait.

A `Compile_Error` is _returned_ as a _value_ and not a panic/dataflow error??? That is so weird. Why is that? This just seems wrong.

Reply:
It's been like that for a while https://github.com/enso-org/enso/blob/develop/engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/TypeOfNode.java#L181

Reply:
Yes, I see that. But that is totally strange.
Can we change it?

Reply:
Sure.

Reply:
I'd say that unknown types in Enso should have type `Any`. Because if you try to call a method on them, you can call any instance method of `Any` and the dispatch is going to work. @kustosz usually advises me when it comes to these `Any`/polyglot issues.